### ISSUE TITLE
Connect AC1715/11

### ISSUE THREAD

**Issue by floabarts** [kongo09/philips-airpurifier-coap#234]
Hello, i can‘t Connect a Model AC1715/11. 
After I have installed the integration from HACS, I always get the following error Message when I try to Configure: Connection to the device not possible. 
Thanks and Greets

**Comment by LaNiSa2016** [comment]
Hello,

Same for me with AC1715/10

**Comment by Paul1865** [comment]
It's not supported.

Also read https://community.home-assistant.io/t/philips-air-purifier/53030/431?page=14 :) .

**Comment by kongo09** [comment]
Yes, I'm sorry. Some AC1715 work, some don't. It depends on your device. Best advice that I have is try to bring this device back to your shop and get a different one.

**Comment by Lecso11** [comment]
> Yes, I'm sorry. Some AC1715 work, some don't. It depends on your device. Best advice that I have is try to bring this device back to your shop and get a different one.

As of now, it seems there is no chance of them working?

**Comment by baulaergrungim** [comment]
I wouldn't count on it. It's probably cuz some models have newer firmware, or something. Phillips now(?) uses Air+ app... I did message them about opening the door a little for HA, stressing out how much more desired it would make their products, and how versatile it would make it, but I bet it won't change much. Unless they get flooded with emails about it, maybe? :) I'm with the same issue on AC175/11, can't connect either via the integration, or the aioairctrl.
